Sources/Networking/Core/Upload/UploadAPIManager.swift

```diff
@@ -93,7 +90,7 @@ extension UploadAPIManager: URLSessionDataDelegate {
 
 // MARK: - URLSessionTaskDelegate
 extension UploadAPIManager: URLSessionTaskDelegate {
-    public func urlSession(
+    nonisolated public func urlSession(
```

Comment:
why do we need to mark it as nonisolated? Would it be same for all delegate methods on own global actors?

Reply:
Yes it would, otherwise the method could not be used to satisfy the delegate protocol requirement.